apps/dev/src/combobox/combobox-demo.component.ts

```diff
@@ -26,43 +26,56 @@ import {
   DtCombobox,
   DtComboboxFilterChange,
 } from '@dynatrace/barista-components/experimental/combobox';
+import { DT_OPTION_CONFIG } from '@dynatrace/barista-components/autocomplete';
 
-const allOptions: { name: string; value: string }[] = [
-  { name: 'Value 3', value: '[value: Value 3]' },
+interface ComboboxDemoOption {
+  readonly name: string;
+  readonly value: string;
+}
+
+const allOptions: ComboboxDemoOption[] = [
+  { name: 'Value 1', value: '[value: Value 1]' },
   { name: 'Value 2', value: '[value: Value 2]' },
-  { name: 'Value 31', value: '[value: Value 31]' },
+  { name: 'Value 3', value: '[value: Value 3]' },
   { name: 'Value 4', value: '[value: Value 4]' },
+  { name: 'Value 5', value: '[value: Value 5]' },
+  { name: 'Value 6', value: '[value: Value 6]' },
+  { name: 'Value 7', value: '[value: Value 7]' },
+  { name: 'Value 8', value: '[value: Value 8]' },
+  { name: 'Value 9', value: '[value: Value 9]' },
+  { name: 'Value 10', value: '[value: Value 10]' },
 ];
 
+function optionFilter(filter: string): (o: ComboboxDemoOption) => boolean {
+  return (option: ComboboxDemoOption): boolean => {
+    return option.value.toLowerCase().indexOf(filter.toLowerCase()) >= 0;
+  };
+}
+
 @Component({
   selector: 'combobox-dev-app-demo',
   templateUrl: 'combobox-demo.component.html',
+  styleUrls: ['combobox-demo.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
+  providers: [{ provide: DT_OPTION_CONFIG, useValue: { height: 40 } }],
```

Comment:
please put the height of `40` in a constant in the top of the file with a description why exactly 40

`const DT_OPTION_DEFAULT_HEIGHT = 40`